openid-connect-server-webapp/src/main/resources/db/tables/loading_temp_tables_oracle.sql

```diff
@@ -0,0 +1,77 @@
+--
```

Comment:
New file (not using `loading_temp_tables.sql`) because:
* same reasons as for `clients.sql`
* additionally, Oracle has a different syntax for creating temporary tables